src/marshmallow/validate.py

```diff
@@ -548,3 +548,25 @@ def __call__(self, value) -> typing.Any:
             if val not in self.choices:
                 raise ValidationError(self._format_error(value))
         return value
+
+
+class ContainsNoneOf(NoneOf):
+    """Validator which fails if ``value`` is a sequence and any element
+    in the sequence is a member of the sequence passed as ``iterable``. Empty input
+    is considered valid.
+
+    :param iterable iterable: Same as :class:`NoneOf`.
+    :param str error: Same as :class:`NoneOf`.
+    """
+
+    default_message = "One or more of the choices you made was in: {values}"
```

Comment:
```suggestion
    default_message = "One or more of the choices you made was in: {values}."
```

for consistency with ContainsOnly